src/elements/helpers/Preview.tsx

```diff
@@ -24,6 +47,28 @@ const getDocHeight = (doc: Document | undefined) => {
   return undefined;
 };
 
+function isUntypedMessageData(data: unknown): data is UntypedMessageData {
```

Comment:
We should aim to use arrow functions consistently within this project.

Reply:
Strongly agree, not sure where that came from. Retro.